Add plugin: Epiphany

artempdev opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [x]  Linux
  - [x]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

artempdev commented:
/skip

Thanks for the feedback! I’m not using FileSystemAdapter directly—just checking if the adapter is an instance of it. I’ve tested on mobile, and no errors occur since it’s only a conditional check. Please review this, and let me know if further changes are needed.

joethei commented:
[async grafanaLog(type: 'log' | 'error', message: string) {](https://github.com/Epiphany-Notes/epiphany-obsidian-plugin/blob/7ee84b1be14da6d180f448d2d73c14d25e0e02b7/main.ts#L46-L64)
Remove this function, as per our [Developer policies](https://docs.obsidian.md/Developer+policies) it's not allowed to include any sort of telemetry in a plugin, that includes error logging to a server.

[this.addSettingTab(new EpiphanySettingTab(this.app, this));](https://github.com/Epiphany-Notes/epiphany-obsidian-plugin/blob/7ee84b1be14da6d180f448d2d73c14d25e0e02b7/main.ts#L387)
There are no settings to display, so this should not be called.

[contentEl.createEl('h2', { text: 'Epiphany Vault Conflict Detected' });](https://github.com/Epiphany-Notes/epiphany-obsidian-plugin/blob/7ee84b1be14da6d180f448d2d73c14d25e0e02b7/main.ts#L456), [text: 'Sync with Existing Vault',](https://github.com/Epiphany-Notes/epiphany-obsidian-plugin/blob/7ee84b1be14da6d180f448d2d73c14d25e0e02b7/main.ts#L463), [text: 'Create New Vault with Different Name',](https://github.com/Epiphany-Notes/epiphany-obsidian-plugin/blob/7ee84b1be14da6d180f448d2d73c14d25e0e02b7/main.ts#L472), [contentEl.createEl('h2', { text: 'Create New Vault' });](https://github.com/Epiphany-Notes/epiphany-obsidian-plugin/blob/7ee84b1be14da6d180f448d2d73c14d25e0e02b7/main.ts#L483), [const submitBtn = contentEl.createEl('button', { text: 'Create Vault' });](https://github.com/Epiphany-Notes/epiphany-obsidian-plugin/blob/7ee84b1be14da6d180f448d2d73c14d25e0e02b7/main.ts#L490)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

artempdev commented:
Hey, thanks for the feedback! I’ve made the following changes:

- Removed the grafanaLog function.
- Removed the settings tab.
- Updated the UI text to use sentence case.

Let me know if anything else is needed!
